Show Current Month When Future Month Start Date is Provided (#28042)
hashicorp/vault: changelog/28042.txt

```release-note:bug

> activity: The sys/internal/counters/activity endpoint will return current month data when the start_date parameter is set to a future date.

Reviewer: ```suggestion
activity: Fixed issue where sys/internal/counters/activity endpoint would not return current month data when the end_date parameter is set to a future date.
```

Reviewer: Some of this is subjective wording, but it should be `end_date`, not `start_date`. Feel free to take just the `end_date` fix
